WalkingHorror.uproject

```diff
@@ -1,13 +1,16 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "5.4",
+	"EngineAssociation": "5.3",
```

Comment:
Did we intend to bump this back down to 5.3?

Reply:
We thought it was causing issues importing dependencies for 'enhancedinput' header files. The Enhanced Input library is UE 5.+ industry standards for handling user inputs. The way it was done before is now deprecated. Turns out we didn't tell the Build.cs files that we needed it. We bumped it down to see if we could import the header files. The superstition was unrewarded. We wanted to bump it to 5.4 to exploit the easy rigging for meshes.

Reply:
Good catch. I think we decided to bump it back up to 5.4 after we confirmed the problem was most likely unrelated to changing it to 5.3. I should be able to make that change and update this PR for version 5.4.